datafusion/physical-plan/src/sorts/sort.rs

```diff
@@ -587,7 +590,13 @@ pub(crate) fn sort_batch(
         .map(|expr| expr.evaluate_to_sort_column(batch))
         .collect::<Result<Vec<_>>>()?;
 
-    let indices = lexsort_to_indices(&sort_columns, fetch)?;
+    let indices = if is_multi_column_with_lists(&sort_columns) {
```

Comment:
FWIW I would anticipate the row format being faster for almost all multi column sorts

Reply:
> FWIW I would anticipate the row format being faster for almost all multi column sorts

My suspicion is that only for large variable length stringss would this approach be faster

Reply:
Certainly when I benchmarked this a while back the cutoff was a lot lower than I was anticipating